Sources/SkyFloatingLabelTextFieldWithIcon.swift

```diff
@@ -11,11 +11,51 @@
 
 import UIKit
 
+/**
+ Identify the type of icon. 
+    - font: Set your icon by setting the font of iconLabel
+    - image: Set your icon by setting the image of iconImageView
+ */
+public enum IconType: Int {
+    case font
+    case image
+}
+
 /**
  A beautiful and flexible textfield implementation with support for icon, title label, error message and placeholder.
  */
 open class SkyFloatingLabelTextFieldWithIcon: SkyFloatingLabelTextField {
 
+    @IBInspectable
+    var iconTypeValue: Int { // swiftlint:disable:this identifier_name
```

Comment:
I think you should remove this Swiftlint disable call. Per the error on Travis

```
	Superfluous Disable Command Violation: SwiftLint rule 'identifier_name' did not trigger a violation in the disabled region. Please remove the disable command. (superfluous_disable_command)
```

Reply:
Nice catch! Thanks!